server_list.go

```diff
@@ -16,19 +16,23 @@ func newServerListCommand(cli *CLI) *cobra.Command {
 		TraverseChildren: true,
 		RunE:             cli.wrap(runServerList),
 	}
+	cmd.Flags().Bool("no-header", false, "Do not print table header")
```

Comment:
Please do not include these changes in this PR.

Let’s discuss separately how we want to allow the user of the CLI to configure how the output should look like (table with headers, table without headers, just selected columns, JSON, YAML, etc.).